pkgs/development/python-modules/hcloud/default.nix

```
@@ -12,11 +12,11 @@
 
 buildPythonPackage rec {
   pname = "hcloud";
-  version = "1.12.0";
+  version = "1.13.0";
```

```suggestion
  version = "1.13.0";
  disabled = isPy27;
```

Instead of in python-packages.nix, disable it here.